### PR TITLE
day 12 proofread

### PR DESCRIPTION
**Pull request by Avuxo**

### CODE REVIEW

File: script/Arcueid/Day 12/12_00_ARC12_2B.txt [hunk 1]
```diff
@@ -66,7 +66,9 @@ The one who smashed the third floor window and came tumbling inside is―――
 [sha:0f11569f373404bf4509036059950659fb50ba2e]{
 -- Page 80, Offset 30766.
 -- 「――――――<Ⅹ|デケム>」
-"――――――Decem."
+// do we want to bring back the X or chi or whatever here?
+// is it intended as a christogram?
+"――――――%{i}Decem.%{/i}"
```
Comment (rschlaikjer) [inline script/Arcueid/Day 12/12_00_ARC12_2B.txt:71]: I thought it was gematria? Petrikow call on whether it's important

File: script/Arcueid/Day 12/12_00_ARC12_2B.txt [hunk 2]
```diff
@@ -248,7 +250,7 @@ Is that... Mario's voice coming out of a speaker...?
 [sha:d3ca6071239ed8302f51af585ed23f8cee803306]{
 -- Page 93, Offset 30801.
 -- 「何者だ。随分と<私|・>に気安いようだが」
-"Who might you be? You seem to be acting awfully friendly with %{i}me%{/i}."
+"Who might you be? You seem to be acting awfully friendly with me."
```
Comment (rschlaikjer) [inline script/Arcueid/Day 12/12_00_ARC12_2B.txt:253]: You could move it to _awfully_ if you want but I'm fine dropping it

**Review by rschlaikjer — APPROVED** [review]
rip yap

File: script/Arcueid/Day 12/12_00_ARC12_2B.txt (hunk 1 above)
Comment (Petrikow) [reply script/Arcueid/Day 12/12_00_ARC12_2B.txt:71]: It's numerology stuff indeed.
I guess you could change it all caps for that ROMANVS feeling but other than that I don't see any reason to change it.

**Pull request merged**